lib/control-connection.js

```diff
@@ -25,10 +25,11 @@ var schemaChangeTypes = {
 /**
  * Represents a connection used by the driver to receive events and to check the status of the cluster
  * @param {Object} options
+ * @param {ProfileManager} profileManager
  * @extends EventEmitter
  * @constructor
  */
-function ControlConnection(options) {
+function ControlConnection(options, profileManager) {
```

Comment:
In the following example:
```javascript
const client = new Client({ 
  contactPoints: ['host1'], 
  profiles: { 
    'default': new ExecutionProfile({
      loadBalancing: myLbPolicy,
      retry: myRetryPolicy
    }) 
  }
});
```
Makes sense to use `myLbPolicy` as default, regardless of the default lbp policy.

If you agree, I'll document it (and remove the unused field).